src/datepicker/datepicker.js

```diff
@@ -221,18 +221,17 @@ export default class Datepicker extends React.Component<
     } else {
       const dateString = this.normalizeDashes(inputValue);
       let date = new Date(dateString);
-      const formatString = this.props.formatString;
-      if (formatString) {
-        // Prevent early parsing of value.
-        // Eg 25.12.2 will be transformed to 25.12.0002 formatted from date to string
-        if (
-          dateString.replace(/(\s)*/g, '').length <
-          formatString.replace(/(\s)*/g, '').length
-        ) {
-          date = null;
-        } else {
-          date = parse(dateString, formatString, new Date());
-        }
+      const formatString = this.props.formatString || 'yyyy/MM/dd';
```

Comment:
I could have extracted `'yyyy/MM/dd'` to something like `DEFAULT_FORMAT_STRING` but it's hardcoded everywhere so I just followed that for consistency

Reply:
Could you add this string as a default value for `formatString`? Then we won't need a conditional assignment.

Reply:
You mean in the `static defaultProps`?

Reply:
correct